src/mappings/reward.ts

```diff
@@ -0,0 +1,73 @@
+import { Address } from '@graphprotocol/graph-ts';
+
+import { ERC20 } from '../../generated/Governance/ERC20';
+import {
+  RewardAdded,
+  RewardPaid,
+  Staked,
+  Withdrawn,
+} from '../../generated/Governance/Governance';
+import { GovernanceRewards } from '../../generated/schema';
+import { BIGINT_ZERO } from '../utils/constants';
+import { getOrCreateAccount, getOrCreateToken } from '../utils/helpers';
+
+let YFI = Address.fromString('0x0bc529c00C6401aEF6D220BE8C6Ea1667F6Ad93e');
+function getOrCreateGovernance(address: Address): GovernanceRewards {
```

Comment:
input on where this should live would be helpful, it doesn't seem to fall into the current files for util as the repo is set up.